### PR TITLE
[bitnami/prometheus-operator] Fix typos in README

### PR DESCRIPTION
**Pull request by xakraz**

### CODE REVIEW

**Comment by marcosbc** [comment]
Also, please update the version of the chart (patch release only, i.e. 0.40.0 -> 0.40.1). Thanks!

**Comment by marcosbc** [comment]
I still see the `maxAvailable` parameter, which AFAIK should be `maxUnavailable`. Were you able to check it?

**Comment by xakraz** [comment]
I have also checked the variables name in the templates and `values.yaml` and `values-production.yaml` :wink:

File: bitnami/prometheus-operator/README.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ The following table lists the configurable parameters of the Prometheus Operator
...
-| `prometheus.podDisruptionBudget.minAvailable`         | Maximum number / percentage of pods that may be made unavailable                                                               | `nil`                                                                                                                                                                                                                                               |
+| `prometheus.podDisruptionBudget.maxUnavailable`         | Maximum number / percentage of pods that may be made unavailable                                                               | `nil`                                                                                                                                                                                                                                               |
```
Comment (marcosbc) [inline bitnami/prometheus-operator/README.md:162]: Could you also fix the allignment of the table?

**Pull request merged**